lib/lotus/repository.rb

```diff
@@ -661,6 +662,21 @@ def exclude(query)
         query.negate!
         query
       end
+
+      # This is a method to check entity persited or not
+      # @param entity
+      # @return true
+
+      def persisted?(entity)
```

Comment:
you can now remove this method, I've just added `Entity#persisted?` in PR https://github.com/lotus/model/pull/174, so you can rebase with master and start using that method